RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_tools.py

```diff
@@ -0,0 +1,115 @@
+import FWCore.ParameterSet.Config as cms
+
+def calibrateReducedEgamma(process):
+    process.load("RecoEgamma.EgammaTools.calibratedEgammas_cfi")
+    process.calibratedPhotons.produceCalibratedObjs = cms.bool(False)
+    process.calibratedElectrons.produceCalibratedObjs = cms.bool(False)
+    process.slimmingTask.add(process.calibratedPhotons)
+    process.slimmingTask.add(process.calibratedElectrons)
+    process.reducedEgamma.applyPhotonCalibOnData = cms.bool(True)
+    process.reducedEgamma.applyPhotonCalibOnMC = cms.bool(False)
+    process.reducedEgamma.applyGsfElectronCalibOnData = cms.bool(True)
+    process.reducedEgamma.applyGsfElectronCalibOnMC = cms.bool(False)
+    process.reducedEgamma.photonCalibEnergySource = cms.InputTag("calibratedPhotons","EGMecalEnergy")
+    process.reducedEgamma.photonCalibEnergyErrSource = cms.InputTag("calibratedPhotons","EGMecalEnergyErr")
+    process.reducedEgamma.gsfElectronCalibEnergySource = cms.InputTag("calibratedElectrons","EGMecalTrkEnergy")
+    process.reducedEgamma.gsfElectronCalibEnergyErrSource = cms.InputTag("calibratedElectrons","EGMecalTrkEnergyErr")
+
+    process.reducedEgamma.gsfElectronCalibEcalEnergySource = cms.InputTag("calibratedElectrons","EGMecalEnergy")
+    process.reducedEgamma.gsfElectronCalibEcalEnergyErrSource = cms.InputTag("calibratedElectrons","EGMecalEnergyErr")
+
+    process.reducedEgamma.gsfElectronPFClusterIsoSources.append(cms.InputTag("calibratedElectrons","EGMscaleUpUncertainty"))
```

Comment:
"IsoSources" is not an appropriate name. Please either update gsfElectronPFClusterIsoSources to somthing like gsfElectronMappedFloats or add a new appropriately named parameter to keep track of the calibrated uncertainties.

Reply:
renamed to FloatValueMap